client/src/components/Flash/flash.css

```diff
@@ -5,7 +5,6 @@
   margin-bottom: 0px;
   padding-top: 3px;
   padding-bottom: 3px;
-  position: fixed;
   width: 100%;
   z-index: 150;
```

Comment:
this has no effect without position property
```suggestion
```

Reply:
Good catch. I'm gonna keep it and add position relative